src/app/submission/components/submit-to-scea/submit-to-scea.component.ts

```diff
@@ -0,0 +1,104 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Project} from "@shared/models/project";
+import {AlertService} from "@shared/services/alert.service";
+import {BrokerService} from "@shared/services/broker.service";
+import {LoaderService} from "@shared/services/loader.service";
+import {SaveFileService} from "@shared/services/save-file.service";
+import {Observable} from "rxjs";
+import {FormGroup, FormControl, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-submit-to-scea',
+  templateUrl: './submit-to-scea.component.html',
+  styleUrls: ['./submit-to-scea.component.css']
+})
+
+export class SubmitToSCEAComponent implements OnInit {
+
+  @Input() project$: Observable<Project>;
+  @Input() submissionEnvelopeId;
+  @Input() submissionEnvelope$;
+  @Input() isSubmitted: boolean;
+  @Input() submissionUrl: string;
+  @Input() isLinkingDone: boolean;
+  @Input() manifest: object;
+  releaseDate: string;
+  project_uuid: string;
+  accession_num: number;
+  curator: string[];
+  experiment_type: string;
+  factor_values: string[];
+  public_release_date: string;
+  hca_update_date: string;
+  study_accession: string;
+  test: string;
+  spreadsheet: any;
+  output_dir: string;
+  zip_format: string;
+
+  sceaForm = new FormGroup({
+    project_uuid: new FormControl('', Validators.required),
```

Comment:
Consider using `ngModel`

Reply:
I need to ask you about this, I'm not sure what the benefit would be based on my current understanding, in this case.